OpenRA.Game/CPos.cs

```diff
@@ -51,7 +51,7 @@ public CPos(int x, int y, byte layer)
 		public static CVec operator -(CPos a, CPos b) { return new CVec(a.X - b.X, a.Y - b.Y); }
 
 		public static bool operator ==(CPos me, CPos other) { return me.Bits == other.Bits; }
-		public static bool operator !=(CPos me, CPos other) { return !(me == other); }
+		public static bool operator !=(CPos me, CPos other) { return me.Bits != other.Bits; }
```

Comment:
This change also didn't show a difference on the sharplab page @teinarss suggested. Both versions resulted in
```
CPos.op_Inequality(CPos, CPos)
    L0000: cmp ecx, edx
    L0002: setne al
    L0005: movzx eax, al
    L0008: ret
```